modules/openapi-generator/src/main/resources/scala-sttp/requests.mustache

```diff
@@ -1,48 +0,0 @@
-{{>licenseInfo}}
```

Comment:
removal of these built-in templates is a breaking change, so this needs to get into 5.0

Reply:
Is compatibility maintained between generators which aren't stable? (sttp generator is marked as beta)